src/backend/Backend.h

```diff
@@ -14,6 +15,7 @@
 class Canvas;
 
 using namespace std;
+using namespace v8;
```

Comment:
Personally would prefer using `v8::Value` in this case...

Reply:
(somewhat related: https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice)

Reply:
Done.